app/src/main/java/com/codepath/selfiespot/models/User.java

```diff
@@ -0,0 +1,28 @@
+package com.codepath.selfiespot.models;
+
+
+import com.parse.ParseClassName;
+import com.parse.ParseObject;
+
```

Comment:
you don't need this class, ParseUser serves the purpose of this - https://parseplatform.github.io/docs/android/guide/#users